_posts/en/2021-08-20-micro-acquisition-pilot-update2.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title:  "Micro-Acquisition Pilot: Status Update 2"
+ref: micro-acquisition-update2
+lang: en
+author: "Rachel Muston, in collaboration with the IT Strategy team and Micro-Acquisition Pilot team members"
+date: "2021-08-20"
+excerpt: "The Micro-Acquisition pilot is live! In this blog post we share what we have learned so far and how we are adjusting and improving."
+---
+<!--markdownlint-disable MD033-->
+Since our last status update, the [Micro-Acquisition website](https://micro-acquisition.alpha.canada.ca/en/index.html) went live and the [first opportunity](https://micro-acquisition.alpha.canada.ca/2021/06/23/building-custom-Azure-marketplace-policies.html) was posted.
+
+As part of the Micro-Acquisition pilot we are trying things, learning and adjusting as we go.
+In preparation for the next opportunity we have made a few changes based on what we learned from the first opportunity.
+
+## Added a mailing list
+
+When we didn't get a lot of followers on our [Twitter account](https://twitter.com/microbuysgc), we did some testing and realized 2 things:
+
+1. It wasn't clear what information would be found by following the Twitter account and
+2. A lot of our target audience is not on Twitter
+
+We now have a mailing list that anyone can subscribe to.
+Through the [mailing list](https://forms-formulaires.alpha.canada.ca/id/36) (and Twitter) we will be sharing when new opportunities are posted and/or updated. We also updated the text on the opportunities page to be more clear what information is available via the Twitter account and mailing list.
+
+## Updated evaluation question
+
+Our first opportunity was for a less well known language (Powershell).
```

Comment:
Was Powershell the cause for people writing answers that didn't pass or is it our hypothesis that the less popular nature of the language may have been a factor in receiving less applications than expected?

If it's a possible factor in less applications than expected, maybe this should be in a different section: Communicating/Advertising to target communities or something like that?

Reply:
I think it was a bit the cause of people writing answers that didn't pass because I think for some folks it was like, I know I don't have that skill but I'm interested and I'll try anyway.  But even still, after re-reading I think that first sentence didn't add to the understanding of this item, so I removed it. :)

Reply:
I wouldn't say that Powershell is a less popular language.  In fact it is used alot in the world of infrastructure.  I would reword to state that it is a "niche" language.

Reply:
Oh I actually took that line out after Guillaume's review as I don't think it added to the point re: the updated evaluation question. So we are all good there.